contracts/src/v0.8/vrf/test/VRFV2Plus.t.sol

```diff
@@ -513,49 +513,56 @@ contract VRFV2Plus is BaseTest {
     assertEq(fulfilled, false);
```

Comment:
q: are the changes in this file intentional?

Reply:
Yeah, done by the VRF team themselves. Foundry updated what block hash was used for tests and VRF relies on the blockhash